README.md

```diff
@@ -209,6 +209,17 @@ Type: `Object`
 
 Sets html attributes and their values on the `menu`. Useful for adding `aria-labelledby` and setting to the value of the `id` attribute on your existing label, to provide context to an assistive technology user.
 
+> [!NOTE]
+>
+> To maintain assistive technology support, menu attributes `id`, `role` and `onMouseLeave` cannot be overridden using `menuAttributes`. Setting `className` will append to the component default and [`menuClasses`](#menuClasses) values.
+
+
+#### `menuClasses` (default: `null`)
+
+Type: `string | null`
+
+Adds custom html classes to the menu elements.
```

Comment:
We just say "menu" elsewhere but should we fix ~elements~ → element, or be specific like `menuAttributes`?

```suggestion
Adds custom html classes to the generated `ul` menu element.
```

Reply:
Updated both here and for `menuAttributes` 😊

Reply:
Even better, thank you